Feature/agnieszka/products

AgnieszkaBaranowska opened the pull request:
section products

ssszafir committed:
update footer-mobile

ssszafir committed:
update footer- mobile

update font-style for adress tag

AgnieszkaBaranowska committed:
html + css products

AgnieszkaBaranowska committed:
html + css products

AgnieszkaBaranowska committed:
html + css products

AgnieszkaBaranowska committed:
Merge branch 'feature/Agnieszka/products' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/Agnieszka/products

MalgorzataMarczynska commented on src/partials/products.html:
```diff
@@ -1 +1,29 @@
-<section class="products">products</section>
\ No newline at end of file
+<section class="products">
```
id dla kotwicy do dodania

MalgorzataMarczynska commented on src/sass/_products.scss:
```diff
@@ -0,0 +1,133 @@
+.products__label{
+    font-family: getFont('font-header');
+    font-style: normal;
```
opcja font-style chyba jest tu nie potrzebna, bo to wartość domyślna

MalgorzataMarczynska commented on src/sass/_products.scss:
```diff
@@ -0,0 +1,133 @@
+    margin-bottom: 130px;
+}
+.products__item--icecream{
+    position: relative;
```
position: relative wrzuć do głównej klasy, nie trzeba będzie jej powtarzać później

MalgorzataMarczynska commented on src/sass/_products.scss:
```diff
@@ -0,0 +1,133 @@
+    font-style: normal;
+    font-weight: getFont('font-regular');
+    font-size: 20px;
+    line-height: 23px;
```
line height zawsze jak line-height(px)/font-size(px) - wynik dzielenia wpisujesz jako line-height

MalgorzataMarczynska commented on src/sass/_products.scss:
```diff
@@ -0,0 +1,133 @@
+}
+.products__item{
+    
+    width: 280px;
```
tu chyba musi być min-width - inaczej będziemy mieli dużo miejsca po bokach jak będziemy rozszerzać

AgnieszkaBaranowska replied:
zostawiłam width bo z min-width rozjeżdżały się kart produktów

MalgorzataMarczynska commented on src/sass/_products.scss:
```diff
@@ -0,0 +1,133 @@
+}
+.products__description{
+    font-family: getFont('font-main');
+    width: 200px;
```
tu też min-width

MalgorzataMarczynska commented on src/sass/_products.scss:
```diff
@@ -0,0 +1,133 @@
+    font-style: normal;
+    font-weight:  getFont('font-bold');
+    font-size: 14px;
+    line-height: 152%;
```
line height lepiej zapisane, ale najlepiej to 1.52

DamianSikorski committed:
wtępne ukończenie skecji advantages

DamianSikorski committed:
wtępne ukończenie skecji advantages

DamianSikorski committed:
wtępne ukończenie skecji advantages

KitsuneAkvma committed:
Reviews/mobile

Skończona sekcja ( bez funkcjonującego slidera), tylko wersja mobile.

KitsuneAkvma committed:
feature/footer poprawka/implementacja zmiennych

Mały fix ze względu na to, że nie ma dzisiaj Kamila

KitsuneAkvma committed:
Merge branch 'feature/advantages' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/advantages

KitsuneAkvma committed:
Advantages with rebase

Damianowi nie działało  to ja wrzucam

kwiatkowsko committed:
Update main.scss

kwiatkowsko committed:
Merge pull request #7 from KitsuneAkvma/feature/footer-mobile

add footer-mobile

DamianSikorski committed:
mała poprawka

DamianSikorski committed:
Merge branch 'feature/advantages' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/advantages

KitsuneAkvma committed:
feature/reviews aktualizacja zmiennych

KitsuneAkvma committed:
Merge branch 'dev' into feature/advantages

kwiatkowsko committed:
Merge pull request #24 from KitsuneAkvma/feature/advantages

add feature advantages-mobile

KitsuneAkvma committed:
feature.reviews/skrócenie kodu

KitsuneAkvma committed:
feature/footer/fix zaktualizowano zmienne

KitsuneAkvma committed:
update footer variables

KitsuneAkvma committed:
MAJOR FIX FOOTER & REVIEW

!!!!!!!!!!!!!!!!!!!!!

kwiatkowsko committed:
Merge pull request #25 from KitsuneAkvma/feature/footer

Feature/footer/patch

kwiatkowsko committed:
publish scss and html contacts

kwiatkowsko committed:
Merge pull request #22 from KitsuneAkvma/feature/reviews

Reviews/mobile fixed

kwiatkowsko committed:
buttons changes

KitsuneAkvma committed:
Merge pull request #26 from KitsuneAkvma/feature/agata/contacts_2

Feature/agata/contacts

KitsuneAkvma committed:
features/reviews/fix

Fixed broken layout ( font size in rood has been changed and it broke whole structure)

kwiatkowsko committed:
Merge pull request #27 from KitsuneAkvma/feature/reviews

pięknie

AgnieszkaBaranowska committed:
update products mobile

AgnieszkaBaranowska committed:
update products mobile

AgnieszkaBaranowska committed:
Merge branch 'feature/Agnieszka/products' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/Agnieszka/products

kwiatkowsko commented:
potrzebna będzie podmiana svg > i kropki, sprawdzenie odstępów pomiędzy kartami, ale to już na nowym branchu. Jest ładnie.
